acceptance-tests/output-recorder/app.rb

```diff
@@ -4,6 +4,13 @@ class App < Sinatra::Base
   EMAIL_RESULT_FILE = '/tmp/email.json'
   JSON_RESULT_FILE = '/tmp/json'
 
+  delete '/teardown' do
+    File.delete(EMAIL_RESULT_FILE)
+    File.delete(JSON_RESULT_FILE)
+
+    status 200
```

Comment:
These files are persisted across test runs?
Always considered these tests immutable, unless you think there is a performance benefit here?

Reply:
Yes the docker continues to run between tests, it's always the same instance. So the data persists across. If I don't teardown/clear.cache it will get the results from the previous test which will result in failure as we're testing the wrong thing.

Reply:
In the future this will need to handle more than 1 email too as we send out multiple emails per submission

Reply:
As we have time left on this card I would like fully test email submissions.